### PR TITLE
fix: add default request timeout

### PR DESCRIPTION
**Pull request by wellrus**

### CODE REVIEW

**Comment by wellrus** [comment]
sometimes request dont get response... we need timeout for retry request

File: lib/config.js [hunk 1]
```diff
@@ -123,5 +123,8 @@ module.exports = rc('license-report', {
 	},
 	httpRetryOptions: {
 		maxAttempts: 5
+	},
+	timeout: {
```
Comment (BePo65) [inline lib/config.js]: I would prefer a name like 'httpTimeoutOptions' to be consistent with 'httpRetryOptions'

File: lib/getPackageJson.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ module.exports = async function(name) {
 
 	const options = {
 		retry: config.httpRetryOptions.maxAttempts,
+		timeout: {
```
Comment (BePo65) [inline lib/getPackageJson.js]: and then this options would become `timeout: config.httpTimeoutOptions`

**Review by BePo65 — CHANGES_REQUESTED** [review]

File: lib/config.js (hunk 1 above)
Comment (wellrus) [reply lib/config.js]: @BePo65 fixed

File: lib/getPackageJson.js (hunk 2 above)
Comment (wellrus) [reply lib/getPackageJson.js]: @BePo65 fixed

**Review by BePo65 — APPROVED** [review]

File: lib/getPackageJson.js (hunk 2 above)
Comment (BePo65) [reply lib/getPackageJson.js]: Thanks for the changes - now the http options are consistent

**Pull request merged**